infrastructure/modules/lambda/main.tf

```
@@ -39,6 +39,19 @@ resource "aws_iam_role_policy" "lambda_role_policy" {
         "s3:GetObject"
       ],
       "Resource": "${var.lambda_s3_bucket_arn}"
+    },
+    {
+      "Effect": "Allow",
+      "Action": [
+        "dynamodb:GetItem",
+        "dynamodb:PutItem"
+      ],
+      "Resource": [
+        "arn:aws:dynamodb:*:*:table/healthcareService",
+        "arn:aws:dynamodb:*:*:table/location",
+        "arn:aws:dynamodb:*:*:table/organisation",
+        "arn:aws:dynamodb:*:*:table/organisationAffiliation"
```

Not sure these are going to work how you want them to, due to the prefix being added to the tables. You'll need
```suggestion
        "arn:aws:dynamodb:*:*:table/${local.prefix}-healthcareService",
        "arn:aws:dynamodb:*:*:table/${local.prefix}-location",
        "arn:aws:dynamodb:*:*:table/${local.prefix}-organisation",
        "arn:aws:dynamodb:*:*:table/${local.prefix}-organisationAffiliation"
```